src/common/GeoRectangularProjection.cc

```diff
@@ -253,7 +253,7 @@ void GeoRectangularProjection::gridLatitudes(const GridPlotting& grid)  const
 	{
 		if ( *lat < -90 ) continue;
 		if ( *lat > 90 ) continue;
-		Polyline poly;
+		magics::Polyline poly;
```

Comment:
It's because win_time.h (our sys/time.h replacement) includes windows.h, which defines a load of things. One of them is a Polyline function, which then clashes with the Magics Polyline function.

In terms of this specific file, it's because we include MatrixHandler.h which includes Timer.h which includes win_time.h.